openedx/core/djangoapps/waffle_utils/views.py

```diff
@@ -108,7 +108,9 @@ def _add_toggle_instance_details(self, toggle, toggle_instance):
         toggle['class'] = toggle_instance.__class__.__name__
         toggle['module'] = toggle_instance.module_name
         if is_code_owner_mappings_configured():
-            code_owner = get_code_owner_from_module(toggle_instance.module_name)
+            code_owner = get_code_owner_from_module(
```

Comment:
You no longer need to call `is_code_owner_mappings_configured`, and `get_code_owner_from_module` is now available from the `monitoring` public API.  See https://github.com/edx/edx-django-utils/blob/master/CHANGELOG.rst#390---2020-10-21  
This is the change I said I didn't know who would get to first.  Seems like it belongs here.